mappings/net/minecraft/nbt/StringNbtReader.mapping

```diff
@@ -16,19 +16,19 @@ CLASS net/minecraft/class_2522 net/minecraft/nbt/StringNbtReader
 	METHOD <init> (Lcom/mojang/brigadier/StringReader;)V
 		ARG 1 reader
 	METHOD method_10716 readComma ()Z
-	METHOD method_10717 parseTagArray ()Lnet/minecraft/class_2520;
+	METHOD method_10717 parseNbtArray ()Lnet/minecraft/class_2520;
 	METHOD method_10718 parse (Ljava/lang/String;)Lnet/minecraft/class_2487;
 		ARG 0 string
 	METHOD method_10719 expect (C)V
-	METHOD method_10721 readCompoundTag ()Lnet/minecraft/class_2487;
-	METHOD method_10722 parseTagPrimitive ()Lnet/minecraft/class_2520;
-	METHOD method_10723 parseTag ()Lnet/minecraft/class_2520;
+	METHOD method_10721 readCompoundNbt ()Lnet/minecraft/class_2487;
+	METHOD method_10722 parseNbtPrimitive ()Lnet/minecraft/class_2520;
+	METHOD method_10723 parseNbt ()Lnet/minecraft/class_2520;
 	METHOD method_10725 readString ()Ljava/lang/String;
-	METHOD method_10726 parseTagPrimitiveArray ()Lnet/minecraft/class_2520;
-	METHOD method_10727 parseCompoundTag ()Lnet/minecraft/class_2487;
+	METHOD method_10726 parseNbtPrimitiveArray ()Lnet/minecraft/class_2520;
+	METHOD method_10727 parseCompoundNbt ()Lnet/minecraft/class_2487;
 	METHOD method_10728 readArray (Lnet/minecraft/class_4614;Lnet/minecraft/class_4614;)Ljava/util/List;
 		ARG 1 arrayTypeReader
 		ARG 2 typeReader
-	METHOD method_10729 parseListTag ()Lnet/minecraft/class_2520;
+	METHOD method_10729 parseListNbt ()Lnet/minecraft/class_2520;
```

Comment:
Those renames do not make sense outside of the other PR.

Reply:
indeed